snippets/country-localization.liquid

```diff
@@ -31,11 +33,11 @@
             {% endif %}
             data-value="{{ country.iso_code }}"
           >
-            <span class="localization-form__currency">
-              {{- country.currency.iso_code }}
-              {{ country.currency.symbol }} |</span
-            >
-            {{ country.name }}
+            <span class="localization-form__disclosure__item-wrapper">
+              <span class="localization-form__country_flag"> {{ country | image_url: width: 14 | image_tag }} </span>
+              <span class="localization-form__currency_code"> {{- localization.country.currency.iso_code }} | {{ country.name }}</span>
+            </span>
+      
```

Comment:
And here I wonder if it could be simplified as well to this and then add a `align-items: center;` to align them vertically.
```suggestion
            <span class="localization-form__disclosure__item-wrapper">
               {{ country | image_url: width: 14 | image_tag }}
              <span class="localization-form__currency"> {{- localization.country.currency.iso_code }} | {{ country.name }}</span>
            </span>
      
```